src/test/java/com/google/gcp_variant_transforms/options/VcfToBqContextTest.java

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Google LLC
+ 
+package com.google.gcp_variant_transforms.options;
+ 
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+ 
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import org.junit.Test;
+ 
+/**
+* Units tests for VcfToBqContext.java
+*/
+public class VcfToBqContextTest {
+  
+  private static final String INPUT_FILE = "sampleInputFile.txt";
+  private static final String OUTPUT = "sampleOutput";
+  private final VcfToBqOptions MOCKED_VCFTOBQ_OPTIONS = create_mockedVcfToBqOptions(INPUT_FILE, OUTPUT);
```

Comment:
Line is longer than 100 chars. Please fix.

https://google.github.io/styleguide/javaguide.html#s4.4-column-limit